webroot/results/admin/check_rounds.php

```diff
@@ -607,7 +607,12 @@ function checkEvents () {
       $competitionOnly = array_diff( $arrayEventsCompetition[$competitionId], $arrayEventsResults[$competitionId] );
       echo "  Old events list: ".implode(' ', $intersect)." <b style='color:#F00'>".implode(' ',$competitionOnly)."</b><br />\n";
       echo "  New events list: ".implode(' ', $intersect)." <b style='color:#3C3'>".implode(' ',$resultsOnly)."</b><br />\n";
-      dbCommand( "UPDATE Competitions SET eventSpecs='".implode(' ', $arrayEventsResults[$competitionId])."' WHERE id='$competitionId'" );
+      foreach($competitionOnly as $event) {
+        dbCommand("DELETE from competition_events where competition_id='$competitionId' and event_id = '$event'");
+      }
+      foreach($resultsOnly as $event) {
+        dbCommand("INSERT INTO competition_events (id, competition_id, event_id) VALUES (NULL, '$competitionId', '$event')");
```

Comment:
Thanks for updating this file, @pingskills! From reading the code, this looks good, but I wanted to confirm if you've actually tried it out, or if we should test it out on staging before we deploy.